olp-cpp-sdk-dataservice-read/tests/ParserTest.cpp

```diff
@@ -441,4 +442,108 @@ TEST(ParserTest, Index) {
   }
 }
 
+TEST(ParserTest, Messages) {
+  {
+    // Testing valid messages array:
+    const std::string valid_message_json =
+        R"jsonString({"metaData":{"partition":"314010583","checksum":"ff7494d6f17da702862e550c907c0a91","compressedDataSize":152417,"dataSize":250110,"data":"iVBORw0KGgoAAAANSUhEUgAAADAAAAAwBAMAAAClLOS0AAAABGdBTUEAALGPC/xhBQAAABhQTFRFvb29AACEAP8AhIKEPb5x2m9E5413aFQirhRuvAMqCw+6kE2BVsa8miQaYSKyshxFvhqdzKx8UsPYk9gDEcY1ghZXcPbENtax8g5T+3zHYufF1Lf9HdIZBfNEiKAAAAAElFTkSuQmCC","dataHandle":"bb76b7747e7523596e74a138b15d92ec","timestamp":1517916706},"offset":{"partition":7,"offset":38562}})jsonString";
+    const std::string invalid_message_json =
+        R"jsonString({"metaData":{"invalid_partition":"314010583","invalid_checksum":"ff7494d6f17da702862e550c907c0a91","invalid_compressedDataSize":152417,"invalid_dataSize":250110,"invalid_data":"iVBORw0KGgoAAAANSUhEUgAAADAAAAAwBAMAAAClLOS0AAAABGdBTUEAALGPC/xhBQAAABhQTFRFvb29AACEAP8AhIKEPb5x2m9E5413aFQirhRuvAMqCw+6kE2BVsa8miQaYSKyshxFvhqdzKx8UsPYk9gDEcY1ghZXcPbENtax8g5T+3zHYufF1Lf9HdIZBfNEiKAAAAAElFTkSuQmCC","invalid_dataHandle":"bb76b7747e7523596e74a138b15d92ec","invalid_timestamp":1517916706},"invalid_offset":{"invalid_partition":7,"invalid_offset":38562}})jsonString";
```

Comment:
It looks great but inconsistent with the rest of the tests in the file. Would you consider adjusting the way these constants are defined?

Reply:
Do you mean the R"jsonString..." style? Actually, this file have both formats (see line 373: https://github.com/heremaps/here-olp-sdk-cpp/blob/master/olp-cpp-sdk-dataservice-read/tests/ParserTest.cpp#L373 ). So, we need to agree on one of those styles.

Reply:
Yes. However, the `"\""` format prevails, and we'd better sticked with it for a while and fixed it in a follow-up commit.

Reply:
Ok, but still we have a lot of other files which are using `R"jsonString"` style. Do you think we need to align all of them to one style?

Reply:
Let's stick to the C++11 R"" style.
It would be nice if we can port others to R"" style also.

Reply:
> Do you think we need to align all of them to one style?
@borzun 

> It would be nice if we can port others to R"" style also.
@andescu

Yes, and yes. 
So I'm ok with any style you pick for this PR as soon as we adapt all files to the single approach of the default choice, that is `R""`